src/org/folio/rest_v2/Authorization.groovy

```diff
@@ -149,4 +149,41 @@ class Authorization extends Common {
             }
         }
     }
+
+    /**
+     * Checks if the tenant is locked by trying to login with the tenant's admin credentials.
+     *
+     * @param tenant The OkapiTenant object representing the tenant.
+     * @return Returns 'true' if the tenant is locked, otherwise returns 'false'.
+     *         If the HTTP response code is other than 404 (NOT FOUND), an exception will be thrown.
+     * @throws RequestException when there's an unexpected error during the REST call.
+     */
+    boolean isTenantLocked(OkapiTenant tenant) {
+        // Construct the URL for the login endpoint.
+        String url = generateUrl("/authn/login")
```

Comment:
As a new feature with RTR will be in place, should we consider using "/authn/login-with-expiry"?
Related doc: https://wiki.folio.org/pages/viewpage.action?pageId=96414255

Reply:
I think we should created additional ticket to implement RTR feature